meta.yaml

```diff
@@ -45,6 +45,7 @@ requirements:
     # Normally installed via pip:
     - cartopy
     - cf-units
+    - cmocean
```

Comment:
It seems the cmocean colormaps are also available in palettable https://jiffyclub.github.io/palettable/cmocean/ . Maybe we don't need both packages?

Reply:
I just need one colormap from palettable, that can actually be generated with matplotlib. Moreover, palettable is not in sync with the latest version of cmocean, so I remove it.